themes/cucumber-sb/static/js/site.js

```diff
@@ -128,27 +136,17 @@ var supportedLanguages = [
   "scala"
 ]
 
-var defaulLanguage = 'java'
+var defaultLanguage = 'java'
 
 ready(function() {
   var selectedLang = getLangFromUrl();
   if((selectedLang == '' || selectedLang == null)){
-    if(localStorage.getItem('language') == ''){
-      showOnly(defaulLanguage)
-      localStorage.setItem('language', defaulLanguage)
-    }else{
-      showOnly(localStorage.getItem('language'))
-    }
+    setDefaultLang()
   }else{
     if(supportedLanguages.includes(selectedLang)){
```

Comment:
Now I think about it, since `null` and `''` are not in the `supportedLanguages` list, maybe we could just merge these two conditionals together? So that we either have a good `selectedLang` and show that one, or we show the "default"

WDYT?

Reply:
@mattwynne ,
you are right! I have made this change, please take a look at it. Thanks :)